### PR TITLE
Small fixes to get CMS Xcache config working

### PR DESCRIPTION
**Pull request by efajardo**
This fixes some lcmaps and other errors we found while installing this rpm in a contianer at UCSD and baremetal at Purdue.

### CODE REVIEW

File: rpm/xcache.spec [hunk 1]
```diff
@@ -197,14 +197,18 @@ mkdir -p %{buildroot}%{_sysconfdir}/grid-security/xrd
...
 %config %{_sysconfdir}/xrootd/config.d/40-cms-xcache-plugin.cfg
-%config %{_sysconfdir}/xrootd/config.d/50-cms-xcache-authz.cfg
 %config %{_sysconfdir}/xrootd/config.d/50-cms-xcache-paths.cfg
 %config(noreplace) %{_sysconfdir}/xrootd/config.d/90-cms-xcache-disks.cfg
 %config(noreplace) %{_sysconfdir}/xrootd/config.d/90-cms-xcache-local-redirector.cfg
 %config(noreplace) %{_sysconfdir}/xrootd/config.d/95-cms-xcache-logging.cfg
 
 %changelog
+* Wed Aug 7 2019 Edgar Fajardo <emfajard@ucsd.edu> - 1.1.0-0.4
+- Changing order of lcmpas configuration for CMS
```
Comment (bbockelm) [inline rpm/xcache.spec]:
```suggestion
- Changing order of LCMAPS configuration for CMS
```

File: configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg [hunk 2]
```diff
@@ -20,5 +20,7 @@ if exec xrootd
    xrd.port 1094
 fi
 
+# This is the configuration for a cache
+all.role server
```
Comment (brianhlin) [inline configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg]: Does this work for the two cases that we want to support?

- A single XCache host
- Multiple caches behind a local redirector

Comment (efajardo) [reply configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg]: Yes role should always be set up. It shoudl not only be set up for redirectors. I have it in all my [standaldone stashcaches](https://github.com/opensciencegrid/prp-stashcache/blob/master/stashcache-server.cfg#L35) so I know it works

File: configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg [hunk 3]
```diff
@@ -20,5 +20,7 @@ if exec xrootd
    xrd.port 1094
 fi
 
+# This is the configuration for a cache
```
Comment (brianhlin) [inline configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg]: Let's just get rid of this comment

```suggestion
```

File: configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg (hunk 2 above)
Comment (brianhlin) [reply configs/cms-xcache/xrootd/xrootd-cms-xcache.cfg]: In that case, could you delete this line? https://github.com/opensciencegrid/xcache/blob/master/configs/cms-xcache/config.d/90-cms-xcache-local-redirector.cfg#L16

**Comment by brianhlin** [comment]
LGTM, go ahead and cut the next RC

**Pull request merged**